lua/nvim-treesitter/install.lua

```diff
@@ -250,25 +246,26 @@ local function do_download_tar(repo, project_name, cache_dir, revision, project_
   if r.exit_code > 0 then
     failed_commands = failed_commands + 1
     finished_commands = finished_commands + 1
-    error(
+    log.error(
       'Error during download, please verify your internet connection: ' .. vim.inspect(r.stderr)
     )
   end
 
-  print('Creating temporary directory')
+  log.info('Creating temporary directory')
+  log.debug('Creating temporary directory: ' .. temp_dir)
   --TODO(clason): use vim.fn.mkdir(temp_dir, 'p') in case stdpath('cache') is not created
   local err = uv_mkdir(temp_dir, 493)
   a.main()
   if err then
     failed_commands = failed_commands + 1
     finished_commands = finished_commands + 1
-    error(string.format('Could not create %s-tmp: %s', project_name, err))
+    log.error('Could not create %s-tmp: %s', project_name, err)
   end
 
-  print('Extracting ' .. project_name .. '...')
+  log.info('Extracting ' .. project_name .. '...')
   r = job.run({
     'tar',
-    '-xvzf',
+    '-xzf',
```

Comment:
When changing the parameters for `tar`, it would be worth checking whether everything still works on FreeBSD. I remember there was one letters added for FreeBSD that was implicit on other OSes (not that I would know what they mean). I'll look for the original PR that changed this for the person to test.

Reply:
`v` is just the verbose flag. If it is implicit on some os's then those users will just have larger logs. We don't actually use stdout from any jobs.

Reply:
You're right. The issue I was referring to added to `-z` flag https://github.com/nvim-treesitter/nvim-treesitter/pull/1598